### PR TITLE
README: Update IRC and contributing link

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,18 +65,19 @@ you may need to run this command with sudo.
...
-We love code contributions. See the contributing.txt file for
+We love code contributions. See the [contributing.md](contributing.md) file for
 details.
 
 
 #### IRC
 
-The irc channel for Meson is `#mesonbuild` over at Freenode.
+The irc channel for Meson is `#mesonbuild` over at 
+[Freenode](https://webchat.freenode.net/#mesonbuild).
```
Comment (nirbheek) [inline README.md]: I am not sure if we should link to the webchat implicitly like that. Maybe add a separate sentence about it?

Comment (nirbheek) [reply README.md]: How about `You can use [FreeNode's official webchat](https://webchat.freenode.net/#mesonbuild) to [...]`?

**Comment by nirbheek** [comment]
Thanks!

**Comment by lzutao** [comment]
You're welcome. It's very pleased to use Meson.

**Pull request merged**